src/makei/init_project.py

```diff
@@ -168,40 +175,88 @@ def create_file(file_path: Path, content: Optional[str], force: bool = False) ->
         file.write(content)
 
 
-def init_project(force: bool = False) -> None:
-    """ A CLI utility to create a project"""
-    signal.signal(signal.SIGINT, _signal_handler)
+def update_json_field(pathToJsonFile: str, updateKey1: str, updateVal: str, updateKey2: str = "") -> None:
```

Comment:
So what are the semantices.  IF the value is "" do we not add the key-value pair?  What about if it is undefined?
Is there a way to delete the field if the `undefined` or empty string is specified
We should really standardize on whether "" or None means unspecified.